pkg/suggestion/nasrl_service.py

```diff
@@ -33,11 +35,13 @@ def __init__(self, request, logger):
         self.search_space = None
         self.opt_direction = None
         self.objective_name = None
+        self.num_trials = 1
```

Comment:
api.GetSuggestionsRequest.request_number has defined num_trials, you need not add a new one in  suggestionParameters
Thanks!

Reply:
Thank you for notice! We will change it.

Reply:
Fixed. I removed num_trials from the suggestion parameters and used requestNumber instead.